bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -56,12 +56,43 @@ parser.add_argument('--max-hierarchical-removal', type=int, default=0,
                     help='Maximum amount of hierarchical removals to carry '
                          'out when hierarchical removal is desired. Choose '
                          '-1 for continuous hiearchical removal until no '
-                         'foreground triggers are louder than the inclusive '
-                         'background. Choose 0 to do no hierarchical '
-                         'removals.  Choose 1 to do at most 1 hierarchical '
-                         'removal, and so on. [default=0]')
+                         'foreground triggers are louder than the '
+                         '(inclusive/exclusive) background. Choose 0 to do '
+                         'no hierarchical removals.  Choose 1 to do at most '
+                         '1 hierarchical removal, and so on. [default=0]')
+parser.add_argument('--hierarchical-removal-against', type=str,
+                                                      default=None,
+                    help='Make a choice to hierarchically remove foreground '
+                         'triggers based on whether it is louder than the '
+                         'inclusive (little-dogs-in) background or the '
+                         'exclusive (little-dogs-out) background. Choose by '
+                         'entering <inclusive> or <exclusive>. '
+                         '[default=inclusive]')
 parser.add_argument('--output-file')
 args = parser.parse_args()
+
+# Check that the user chose inclusive or exclusive background to perform
+# hierarchical removals of foreground triggers against.
+
+if args.max_hierarchical_removal == 0:
+    if args.hierarchical_removal_against is not None:
+        parser.error("Cannot choose to do no hierarchical removals and " \
```

Comment:
This error is confusing. I would just say:

You must specify the maximum number of removals with --max-hierarchical-removal if the argument --hierarchical-removal-against is given.

Reply:
Honestly, we don't really need this error. `--hierarchical-removal-against` becomes irrelevant if `--max-hierarchical-removal 0`.